frontend/src/metabase/components/AccordianList.jsx

```diff
@@ -164,91 +198,151 @@ export default class AccordianList extends Component {
         return this.props.getItemClasses && this.props.getItemClasses(item, itemIndex);
     }
 
+    renderSectionHeader(section, sectionIndex) {
```

Comment:
Would probably be good to remove this if it's not called anywhere (doesn't look like it is).

Reply:
Done.